packages/airbnb-base/README.md

```diff
@@ -168,64 +168,26 @@ module.exports = {
   use: [
     ['@neutrinojs/airbnb-base', {
       eslint: {
-        rules: {
```

Comment:
It's worth noting that this form still works, it's just not what we're using for the defaults / what we're recommending. Any preset that pass in the old form will still correctly override the values under `baseConfig`, since ESLint's CLIEngine (plus our eslintrc conversion function) merges them in such a way that the top-level options take precedence.